product_supplierinfo_revision/README.rst

```diff
@@ -23,8 +30,9 @@ To use this module:
 #. By clicking on Apply button, a new supplier pricelist will be created with
    a price increased accordingly to the percent entered. Old supplier
    pricelist will have a date end from which pricelist will be out of date.
-#. You can check supplier pricelists as well as variation percent in the
-   product form, by clicking on *Purchase* tab.
+#. Once the Inventory Management (*stock*) module is installed, you can check 
+   supplier pricelists as well as variation percent in the product form, by 
+   clicking on *Inventory* tab.
```

Comment:
On client instance I look suppliers in Purchase tab, but if you has check...

Reply:
I don't remember exactly but I think that the location depends on installed modules. 
I've checked again and if Purchase Management is install, suppliers are located in product's Purchase tab. I will look further, don't merge.